api.js

```diff
@@ -25,4 +25,20 @@ export const getArticlesByID = (id) => {
     })
 }
 
+export const updateArticleVotes = (id, action) => {
```

Comment:
You could refactor this to just take the vote amount and then you'd have no need for conditional logic. But it does work nicely, just a suggestion.